server/uno.go

```diff
@@ -150,26 +151,26 @@ func drawCard(game string, username string) bool {
 
 // TODO: need to deal the actual cards, not just random numbers
 func dealCards() {
-    // The game has started, no more players are joining
-    // loop through players, set their cards
-    gameStarted = true
-    currPlayer = players[rand.Intn(len(players))]
-    deck := generateShuffledDeck()
-
-    for k := range players {
-        cards := []model.Card{}
-        for i := 0; i < 7; i++ {
-
-            drawnCard := deck[len(deck)-1]
-            deck = deck[:len(deck)-1]
-            cards = append(cards, drawnCard)
-            //cards = append(cards, model.Card{rand.Intn(10), randColor(rand.Intn(4))})
-        }
-        allCards[players[k]] = cards
-    }
-
-    currCard = deck
-    //currCard = newRandomCard()
+	// The game has started, no more players are joining
+	// loop through players, set their cards
+	gameStarted = true
```

Comment:
So. I think it's important to note that all of these methods were storing the current player via internal structs... where it is going to eventually all be stored in the database.  I think the actual-uno-deck team has a solution that resolves turn order & dealing cards and stores it in the database.  

For the meantime, if you want to remove these changes on uno.go, and just keep the ones related to the current player color. I think that change makes sense, and I would leave the other changes to that team.

Reply:
hmm... in retrospect. I went to go check out the branch related to actual-uno-deck, and I can't find their branch.. so I may have been wrong on that change.

Reply:
For now I will remove the changes I made to uno.go. If it turns out that no group is working to fix this issue during the first sprint, we can make it an issue to handle during the second sprint. Since this is such a minor/simple issue to fix, I don't think it hurts anything to handle it later. Does that sound like a good idea?

Reply:
I think that makes good sense to me. :)

Reply:
@brent-buffenbarger can you do me a quick favor. The changes seem appropriate to me. Can you just post a screenshot as a comment of the behavior on master vs the behavior on your branch?  Once that is posted, I'll merge this in.

Reply:
@logan-hendricks I have added images above, Thanks!

Reply:
@brent-buffenbarger thanks for doing that! I'm surprised on Players 2 turn. it doesn't appear to be highlighting. Is that correct?

Reply:
@logan-hendricks No it will highlight Player 2 whenever it is their turn. The top 2 images are based on the update coming from this pull request showing what it looks like when it is Player 1's turn from Player 1's screen and Player 2's screen. The bottom 2 images are from master where this update has not been implemented, so those are showing that before this update there was no indication of whose turn it currently is. Sorry if I made that confusing.

Reply:
ha ha. you're good. that makes more sense. That looks appropriate. I'll merge this now.